packages/kbn-test/src/functional_test_runner/lib/snapshots/decorate_snapshot_ui.test.ts

```diff
@@ -135,10 +135,7 @@ describe('decorateSnapshotUi', () => {
     let lifecycle: Lifecycle;
     beforeEach(() => {
       lifecycle = new Lifecycle();
-      const ci = process.env.CI;
-      process.env.CI = 'true';
-      decorateSnapshotUi(lifecycle, true);
-      process.env.CI = ci;
+      decorateSnapshotUi(lifecycle, false, true);
```

Comment:
Would you mind switching to an options object so that we can avoid naked boolean arguments that are hard to read in a context like this?